client/src/routes/settings/+page.svelte

```diff
@@ -21,11 +54,165 @@
       errorMsg = err instanceof Error ? err.message : String(err);
     }
   }
+
+  function linkGoogleAccount() {
+    try {
+      linkWithGoogle();
+      googleLinked = true;
+      return;
+    } catch (err) {
+      errorMsg = err instanceof Error ? err.message : String(err);
+      outputErrMsg();
+      return;
+    }
+  }
+
+  function linkMicrosoftAccount() {
+    try {
+      linkWithMicrosoft();
+      microsoftLinked = true;
```

Comment:
This displays I successfully linked microsoft account even though i only clicked the button.